sylt/src/formatter.rs

```diff
@@ -454,19 +480,20 @@ fn write_statement<W: Write>(dest: &mut W, indent: u32, statement: Statement) ->
             }
             write_expression(dest, indent, value)?;
         }
-        StatementKind::EmptyStatement => (),
+        StatementKind::EmptyStatement => {},
```

Comment:
Either `(),` or `{}` but not both. Personally, I prefer `(),` since it's on one line.